### PR TITLE
Add Gross Enrolment Rate report

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Changes:

- Added GER graph
- District (aka sub_district) level only currently (See refinement notes on ticket for explanation)

---

### Screenshots:
See attached ticket

### CODE REVIEW

File: packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    {
+      transform: 'select',
+      "'sort_order'":
+        "translate($row.name, { 'Primary': 1', 'Lower Secondary': '2', 'Upper Secondary': '3' })",
+      '...': '*',
+    },
+    {
+      transform: 'sort',
+      by: '$row.sort_order',
+    },
```
Comment (edmofro) [inline packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js:77]: Would be a nice tweak to drop sort_order back off afterwards through an extra select, just so the front end isn't getting clutter

**Review by edmofro — APPROVED** [review]
lgtm! one optional suggestion

File: packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    {
+      transform: 'select',
+      '...': ['name', 'Male', 'Female', 'Total', 'GPI'],
+    },
+  ],
+};
+
+const FRONT_END_CONFIG = {
+  name: 'Gross Enrolment Rate - GER (by grade and gender, GPI)',
```
Comment (edmofro) [inline packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js:86]:
```suggestion
  name: 'Gross Enrolment Rate - GER (by level of education and gender, GPI)',
```

File: packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+  type: 'chart',
+  chartType: 'composed',
+  xName: 'Level of Education',
```
Comment (edmofro) [inline packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js:89]:
```suggestion
  xName: 'Level of Education',
  yName: 'Rate (%),
```

Comment (EMcQ-BES) [reply packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js:89]: LESMIS currently doesn't display `yName` at all, but putting the `yName` at this level applies it to _both_ yAxis in Tupaia, currently I have the `yName` under the `GPI` and `Male` individual configs for the sake of that

image

Comment (edmofro) [reply packages/database/src/migrations/20210623030624-LESMISGrossEnrolmentRate-modifies-data.js:89]: Aaah right that makes sense! I'll fix mine up to be the same

**Comment by edmofro** [comment]
Closing in favour of https://github.com/beyondessential/tupaia/pull/2710

**Comment by edmofro** [comment]
Erm I think this got reopened by zenhub, should definitely be closed

**Pull request closed without merging**